### PR TITLE
Golint fixed for modules/setting

### PR DESCRIPTION
**Pull request by lunny**

### CODE REVIEW

File: modules/setting/setting.go [hunk 1]
```diff
@@ -18,10 +18,10 @@ import (
 	"time"
 
 	"github.com/Unknwon/com"
-	_ "github.com/go-macaron/cache/memcache"
+	_ "github.com/go-macaron/cache/memcache" // memcahe plugin for cache
```
Comment (tboerger) [inline modules/setting/setting.go]: Typo

File: modules/setting/setting.go [hunk 2]
```diff
@@ -30,44 +30,51 @@ import (
...
+// enumerates all the scheme types
 const (
-	HTTP        Scheme = "http"
-	HTTPS       Scheme = "https"
-	FCGI        Scheme = "fcgi"
-	UNIX_SOCKET Scheme = "unix"
+	HTTP       Scheme = "http"
+	HTTPS      Scheme = "https"
+	FCGI       Scheme = "fcgi"
+	UNIXSOCKET Scheme = "unix"
```
Comment (tboerger) [inline modules/setting/setting.go]: Why not UnixSocket?

**Comment by lunny** [comment]
Done. @tboerger

**Comment by bkcsoft** [comment]
Why did you change
```
Foo,  Bar string

Into

Foo string
Bar string
```
?

**Comment by lunny** [comment]
It's golint's indication. 👎

**Comment by strk** [comment]
LGTM (although I'd have preferred smaller PR)

**Comment by thibaultmeyer** [comment]
LGTM

**Comment by lunny** [comment]
resolved #70

**Pull request merged**